BitwardenShared/Core/Platform/Services/StateServiceTests.swift

```diff
@@ -1817,34 +1814,32 @@ class StateServiceTests: BitwardenTestCase { // swiftlint:disable:this type_body
 
         appSettingsStore.syncToAuthenticatorByUserId["1"] = true
 
-        var publishedValues = [SyncToAuthenticatorValue]()
+        var publishedValues = [(userId: String?, shouldSync: Bool)]()
         let publisher = await subject.syncToAuthenticatorPublisher()
             .sink(receiveValue: { userId, shouldSync in
-                publishedValues.append(SyncToAuthenticatorValue(userId: userId, shouldSync: shouldSync))
+                publishedValues.append((userId: userId, shouldSync: shouldSync))
             })
         defer { publisher.cancel() }
 
         try await subject.setSyncToAuthenticator(false)
 
-        XCTAssertEqual(
-            publishedValues,
-            [
-                SyncToAuthenticatorValue(userId: "1", shouldSync: true),
-                SyncToAuthenticatorValue(userId: "1", shouldSync: false),
-            ]
-        )
+        XCTAssertEqual(publishedValues[0].userId, "1")
+        XCTAssertEqual(publishedValues[0].shouldSync, true)
+        XCTAssertEqual(publishedValues[1].userId, "1")
+        XCTAssertEqual(publishedValues[1].shouldSync, false)
```

Comment:
🎨 You can use `==` operator to reduce this a bit more:
```suggestion
        XCTAssertTrue(publishedValues[0] == (userId: "1", shouldSync: false))
        XCTAssertTrue(publishedValues[1] == (userId: "1", shouldSync: true))
```

Reply:
Ah, good call. Update all of them to this style in latest. 👍

Reply:
Sorry, didn't see @KatherineInCode's comment when I updated. Let me know which one y'all prefer. I'm good either way.

Reply:
@brant-livefront Let's go with the `XCTAssertEquals` approach. Depending on how often we need to check the tuple in tests, [a custom assertion function](https://qualitycoding.org/swift-test-assertions/) might also be helpful.

Reply:
@KatherineInCode Sounds good, I'll roll back to using explicit `XCTAssertEquals`.

Reply:
@KatherineInCode this is updated in the latest commit. 👍